ci-operator/config/openshift/knative-client/openshift-knative-client-release-next__4.10.yaml

```diff
@@ -59,6 +65,20 @@ images:
       - destination_dir: .
         source_path: /go/bin/kn
   to: knative-client
+- dockerfile_literal: |-
+    FROM src
+    RUN yum install -y httpd-tools
+    RUN wget -q https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq &&\
+      chmod +x /usr/bin/yq
+    RUN ln -s /usr/bin/oc /usr/bin/kubectl &&\
+      chmod g+rw /etc/passwd
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: test-e2e
```

Comment:
I've used `dockerfile_literal` for a testing purpose. That can be still stored among other Dockerfiles.

Reply:
`dockerfile_literal` seems very verbose... to inline this :man_shrugging: 
Not sure I like it

Reply:
It was a debugging approach to get the bits dialed in. It's superseded by https://github.com/openshift/knative-client/commit/7ff2766b97d2e37908a468de9a492fac6dea81ee now